public/videoplayer.js

```diff
@@ -2,33 +2,33 @@ var canvas_container = document.getElementById("canvas-container");
 var placeholder_video_id = 'KRUbhRKiauA'; //intro to cci 2020 video
 
 var vid_id = {
-  alex: '',
-  bea: 'n1W8W9TUKfY', //example video on personal channel, will need changing to cci youtube
-  cat: '',
-  jann: '',
-  dejana: '',
-  ed: '',
-  ella: '',
-  eunah: '',
-  jessie: 'LGQV5g9hyGE', //example video on personal channel, will need changing to cci youtube
-  kamil: '',
-  khalisha: '',
-  kirsty: '',
-  laiqa: '',
-  lexy: '',
-  llorenc: '',
-  martin: '',
-  max: '',
-  mia: '',
-  morgan: '',
-  riya: '',
-  rukiah: '',
-  sandro: '',
-  stacy: '',
-  tom: '',
-  val: '',
-  vince: '',
-  zhiqin: ''
+  alex: 'oy0rQh7b9Vk',
+  bea: 'a6fU21hJac4', //example video on personal channel, will need changing to cci youtube
```

Comment:
```suggestion
  bea: 'a6fU21hJac4',
```
delete obsolete comment